### DISCUSSION TITLE
Using share for music location

### DISCUSSION THREAD

**Discussion by tunfie** [swingmx/swingmusic#218] [Q&A]
My music is on an external share (nextcloud). I can only select home or music folder under my setup (casaos docker install). Is there an option or feature request where i can add a webdav link or so for my mysic location so i can seperated all software from data?

**Comment by unknwn1993** [comment]
You should be able to Mount the webdav in the Main Music Folder in Casaos, so Swing can handle His content Like local files.  But If you do so, the delay increases cause the song needs to Download twice. 

To Mount IT you need to use he shell AS root.
